.gitignore

```diff
@@ -1,3 +1,4 @@
+built/
 node_modules/
 selenium/
 testapp/inbrowsertest/
```

Comment:
I think we should we add lib/ to `.npmignore` as well, right? Or will TypeScript users need that for typing info?